docs/guide/sections/dom-binding.md

```diff
@@ -1,6 +1,71 @@
-Full documentation for the DOM Binding module is coming soon.
+Cart.js ships with support for [Rivets.js][], which allows for some powerful templating approaches with minimal effort. Rivets.js allows you to describe in your markup how your data should be rendered, and then handles automatically updating the DOM when that model data changes. With Cart.js, this allows us to describe a template for our cart that will be automatically updated whenever the cart changes.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+You can see an example of this in action [at the end of this section].
```

Comment:
I was thinking interactive, but maybe that's overkill. Code block would be fine too :).